src/View/TemplatePaths.php

```diff
@@ -258,14 +258,14 @@ public function resolveTemplateFileForControllerAndActionAndFormat($controller,
 	}
 
 	/**
-	 * @param string $controllerName
+	 * @param string|NULL $controllerName
 	 * @param string $format
 	 * @return array
 	 */
 	public function resolveAvailableTemplateFiles($controllerName, $format = self::DEFAULT_FORMAT) {
```

Comment:
if it's optional anyway, lets make it optional in the signature as well :) ```resolveAvailableTemplateFiles(NULL)``` feels stupid